code/game/turfs/open/_open.dm

```diff
@@ -261,3 +263,11 @@
 		air.set_moles(GAS_CO2, max(air.get_moles(GAS_CO2)-(pulse_strength/1000),0))
 		air.set_moles(GAS_O2, max(air.get_moles(GAS_O2)-(pulse_strength/2000),0))
 		air.adjust_moles(GAS_PLUOXIUM, pulse_strength/4000)
+
+/turf/open/IgniteTurf(power, fire_color)
+	if(turf_fire)
+		turf_fire.AddPower(power)
+		return
+	if(isopenspaceturf(src) || isspaceturf(src))
```

Comment:
use isgroundlessturf, you probably dont need isopenspaceturf defined if you do that

Reply:
isgroundlessturf